### PR TITLE
Clarify versioning of CIS benchmark assessment and hardening guides

### PR DESCRIPTION
**Pull request by catherineluse**
I made this PR after discussing with @paraglade about how to make the versioning of our security hardening guides less confusing.

### CODE REVIEW

File: content/rancher/v2.x/en/security/_index.md [hunk 1]
```diff
@@ -45,24 +45,35 @@ For details, refer to the section on [security scans.]({{<baseurl>}}/rancher/v2.
...
-- [Hardening Guide for Rancher v2.3.3+ with Kubernetes 1.16]({{< baseurl >}}/rancher/v2.x/en/security/hardening-2.3.3/)
+The hardening guide provides prescriptive guidance for hardening a production installation of Rancher v2.1.x, v2.2.x and v.2.3.x. See Rancher's [Self Assessment of the CIS Kubernetes Benchmark](#cis-benchmark-rancher-self-assessment) for the full list of security controls.
+
+> The hardening guides describe how to secure the nodes in your cluster, and it is recommended to follow a hardening guide before installing Kubernetes.
+
+Each version of the hardening guide is intended to be used with specific versions of the CIS Kubernetes Benchmark, Kubernetes, and Rancher:
+
+Hardening Guide Version | Rancher Version | CIS Benchmark Version | Kubernetes Version
+------------------------|----------------|-----------------------|------------------
+[Hardening Guide v2.1]({{<baseurl>}}/rancher/v2.x/en/security/hardening-2.1/) | Rancher v2.1.x | Benchmark v1.3.0 | Kubernetes 1.11
```
Comment (paraglade) [inline content/rancher/v2.x/en/security/_index.md]: I am thinking the latest version of rancher should be at the top of the table

File: content/rancher/v2.x/en/security/benchmark-2.3.3/_index.md [hunk 2]
```diff
@@ -3,18 +3,18 @@ title: CIS Benchmark Rancher Self-Assessment Guide - Rancher v2.3.3+
 weight: 103
```
Comment (paraglade) [inline content/rancher/v2.x/en/security/benchmark-2.3.3/_index.md]: title on this page should have the`+` removed

File: content/rancher/v2.x/en/security/_index.md [hunk 3]
```diff
@@ -45,24 +45,35 @@ For details, refer to the section on [security scans.]({{<baseurl>}}/rancher/v2.
...
-* [CIS Kubernetes Benchmark 1.4.1 - Rancher 2.2.x with Kubernetes 1.13]({{< baseurl >}}/rancher/v2.x/en/security/benchmark-2.2/#cis-kubernetes-benchmark-1-4-1-rancher-2-2-x-with-kubernetes-1-13)
-* [CIS Kubernetes Benchmark 1.4.1 - Rancher 2.3.0-2.3.2 with Kubernetes 1.15]({{< baseurl >}}/rancher/v2.x/en/security/benchmark-2.3/#cis-kubernetes-benchmark-1-4-1-rancher-2-3-0-2-3-2-with-kubernetes-1-15)
-* [CIS Kubernetes Benchmark 1.4.1 - Rancher 2.3.3+ with Kubernetes 1.16]({{< baseurl >}}/rancher/v2.x/en/security/benchmark-2.3.3/#cis-kubernetes-benchmark-1-4-1-rancher-2-3-3-+-with-kubernetes-1-16)
+Each version of Rancher's self assessment guide corresponds to specific versions of the hardening guide, Rancher, Kubernetes, and the CIS Benchmark:
+
+Self Assessment Guide Version | Rancher Version | Hardening Guide Version | Kubernetes Version | CIS Benchmark Version
+---------------------------|----------|---------|-------|-----
+[Self Assessment Guide v2.1]({{<baseurl>}}/rancher/v2.x/en/security/benchmark-2.1/) | Rancher v2.1.x | Hardening Guide v2.1 | Kubernetes 1.11 | Benchmark 1.3.0
```
Comment (paraglade) [inline content/rancher/v2.x/en/security/_index.md]: I am thinking the latest version of rancher should be at the top of the table

File: content/rancher/v2.x/en/security/_index.md (hunk 1 above)
Comment (catherineluse) [reply content/rancher/v2.x/en/security/_index.md]: I rearranged the rows in order from newest version to oldest version

File: content/rancher/v2.x/en/security/_index.md (hunk 3 above)
Comment (catherineluse) [reply content/rancher/v2.x/en/security/_index.md]: I rearranged the rows in order from newest version to oldest version

File: content/rancher/v2.x/en/security/benchmark-2.3.3/_index.md (hunk 2 above)
Comment (catherineluse) [reply content/rancher/v2.x/en/security/benchmark-2.3.3/_index.md]: Removed the +

File: content/rancher/v2.x/en/cluster-provisioning/node-requirements/_index.md [hunk 4]
```diff
@@ -176,3 +177,9 @@ However, this traffic may be blocked when:
 - You are using nodes that have multiple interfaces (multihomed).
 
 In these cases, you have to explicitly allow this traffic in your host firewall, or in case of public/private cloud hosted machines (i.e. AWS or OpenStack), in your security group configuration. Keep in mind that when using a security group as source or destination in your security group, explicitly opening ports only applies to the private interface of the nodes/instances.
+
+# Optional: Security Considerations
+
+If you want to provision a secure Kubernetes cluster that is compliant with the CIS (Center for Internet Security) Kubernetes Benchmark, we recommend to following our hardening guide to secure your nodes before installing Kubernetes.
```
Comment (cloudnautique) [inline content/rancher/v2.x/en/cluster-provisioning/node-requirements/_index.md]: drop the word secure.

Comment (catherineluse) [reply content/rancher/v2.x/en/cluster-provisioning/node-requirements/_index.md]: Dropped the word secure https://github.com/rancher/docs/pull/2208

**Pull request merged**